test/Serilog.Settings.Configuration.Tests/ConfigurationReaderTests.cs

```diff
@@ -14,7 +14,7 @@ public class ConfigurationReaderTests
         public ConfigurationReaderTests()
         {
             _configurationReader = new ConfigurationReader(
-                JsonStringConfigSource.LoadSection(@"{ 'Serilog': {  } }", "Serilog"),
+                JsonStringConfigSource.LoadSection(@"{ ""Serilog"": {  } }", "Serilog"),
```

Comment:
[This](https://github.com/serilog/serilog-settings-configuration/pull/203/files?file-filters%5B%5D=.cs&file-filters%5B%5D=.md&file-filters%5B%5D=.png&file-filters%5B%5D=.ps1&file-filters%5B%5D=.yml#diff-7bebd3e3fb423271f72e33bd8524a4e4R14) may be helpful. Still waiting for merge :(

Reply:
i did not saw that PR ;)
Also the double quote seems fine for all TFM, do we want to have different JSON delimiter since one works for all ?

Reply:
My interest was to keep the number of changes in PR to a minimum. I achieved this with a few lines of code instead of a lot of refactoring. As a result, the tests work on both platforms. In addition, you will simply stop testing a single quote syntax. This is degradation of the test coverage.